addons/link_tracker/models/link_tracker.py

```diff
@@ -42,7 +42,7 @@ class LinkTracker(models.Model):
     label = fields.Char(string='Button label')
     # Tracking
     link_code_ids = fields.One2many('link.tracker.code', 'link_id', string='Codes')
-    code = fields.Char(string='Short URL code', compute='_compute_code')
+    code = fields.Char(string='Short URL code', compute='_compute_code', inverse="_inverse_code", readonly=False)
```

Comment:
As it is by essence computed on a 2many, changing it into an inverse is really strange...

Reply:
Hello @tde-banana-odoo :smile: 

lna would like to have the code editable through the back end (the `KA1`)
![image](https://github.com/user-attachments/assets/0ba58e6a-c3ad-46d0-8928-dd82b7611c6b)

But as it is computed on a one2many I'm not sure how to handle it :thinking: 
It was previously always taking the most recent `link.tracker.code` record code:
```python
def _compute_code(self):
    for tracker in self:
        record = self.env['link.tracker.code'].search([('link_id', '=', tracker.id)], limit=1, order='id DESC')
        tracker.code = record.code
```

So I suppose we always want the most recent code or, if explicitly changed, use the one provided by the user in the form (use the existing code or create a new `link.tracker.code` record for the given code if it doesn't exist).

Do you have any idea on how to achieve that? :confused: 
With my current implementation, if I set an existing code in the form it will take back the most recent one..

Reply:
Mmmh ... ok-ish I guess. But inverse is not updating the last one, is it ?

Reply:
Oh yes I guess I could do something like that!

```python
def _compute_code(self):
    for tracker in self:
        record = self.env['link.tracker.code'].search([('link_id', '=', tracker.id)], limit=1, order='id DESC')
        tracker.code = record.code

def _inverse_code(self):
    self.ensure_one()
    record = self.env['link.tracker.code'].search([('link_id', '=', self.id)], limit=1, order='id DESC')
    record.code = self.code
```

Does this look better ? :thinking: :smile:

Reply:
Looks better to me (just not sure we always have a code, be defensive about it :) )

Reply:
Once updated would suggest a new testing round from Luc, as there were some changes, better be sure :)